python-avd/pyavd/_cv/workflows/verify_devices_on_cv.py

```diff
@@ -4,35 +4,46 @@
 from __future__ import annotations
 
 from logging import getLogger
+from typing import TYPE_CHECKING
+
+from pyavd._cv.client.exceptions import CVResourceNotFound
 
-from ..client import CVClient
-from ..client.exceptions import CVResourceNotFound
 from .models import CVDevice
 
+if TYPE_CHECKING:
+    from pyavd._cv.client import CVClient
+
 LOGGER = getLogger(__name__)
 
 
-async def verify_devices_on_cv(devices: list[CVDevice], workspace_id: str, skip_missing_devices: bool, warnings: list[Exception], cv_client: CVClient) -> None:
-    """
-    Verify that the given Devices are already present in the CloudVision Inventory & I&T Studio.
-    """
+async def verify_devices_on_cv(
+    *, devices: list[CVDevice], workspace_id: str, skip_missing_devices: bool, warnings: list[Exception], cv_client: CVClient
```

Comment:
do we want to force all args to be keywords only?

Reply:
It makes it easier to work with if all of them are kwargs.